hack/quickstart/quickstart-aws.md

```diff
@@ -60,20 +60,24 @@ $ IDENT=k8s-key.pem ./init-master.sh <PUBLIC_IP>
 After the master bootstrap is complete, you can continue to add worker nodes. Or cluster state can be inspected via kubectl:
 
 ```
-$ kubectl --kubeconfig=cluster/auth/kubeconfig get nodes
+$ kubectl --kubeconfig=cluster/auth/admin-kubeconfig get nodes
 ```
 
 ### Add Workers
 
 Run the `Launch Nodes` step for each additional node you wish to add, then using the public-ip, run:
 
 ```
-IDENT=k8s-key.pem ./init-worker.sh <PUBLIC_IP> cluster/auth/kubeconfig
+IDENT=k8s-key.pem ./init-worker.sh <PUBLIC_IP>
 ```
 
-**NOTE:** It can take a few minutes for each node to download all of the required assets / containers.
+**NOTE:**
+- If the master has been bootstrapped over an hour ago, each worker will have
+ to be [approved](../auto_approver.sh) manually.
```

Comment:
I may either link https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/ (look at the last section) or https://github.com/gtank/csrctl, whichever you think is more meaningful.